test/mocks/upstream/cluster_info.h

```diff
@@ -28,7 +28,7 @@ class MockClusterInfo : public ClusterInfo {
   MOCK_CONST_METHOD0(connectTimeout, std::chrono::milliseconds());
   MOCK_CONST_METHOD0(perConnectionBufferLimitBytes, uint32_t());
   MOCK_CONST_METHOD0(features, uint64_t());
-  MOCK_CONST_METHOD0(httpCodecOptions, uint64_t());
+  MOCK_CONST_METHOD0(http2Settings, const Http::Http2Settings&());
```

Comment:
thank you @mattklein123 , i fixed the integration tests failures by add ON_CALL.